tests/integration/helpers/settings.bash

```diff
@@ -1,3 +1,4 @@
 user=admin
 NC_FEED="https://nextcloud.com/blog/static-feed/"
-HEISE_FEED="https://www.heise.de/rss/heise-atom.xml"
\ No newline at end of file
+HEISE_FEED="https://www.heise.de/rss/heise-atom.xml"
+NO_GUID_FEED="https://github.com/nextcloud486153/nextcloud-app-news/blob/master/tests/integration/feeds/no_guid_feed.xml"
```

Comment:
This still needs to be changed after the merge (or beforehand, but then the check won't succeed).
Also, a raw.githubusercontent.com URL should be used there (for this repo it would be https://raw.githubusercontent.com/nextcloud/news/master/tests/integration/feeds/no_guid_feed.xml)

Reply:
I've changed it to my fork, which should still work after merging. I could open a second pull request right after this one to change the url to the nextcloud/news repo. Is that ok?